inference results for 01.04.24

bkotzen opened the pull request:
hey @martinjankowiak here are new bvas results! Happy new year :)

bkotzen committed:
inference results for 01.04.24

bkotzen committed:
inference results for 01.04.24

martinjankowiak commented:
@bkotzen thanks although it looks like you have 01.05 in various places where i'd expect 01.04

bkotzen commented:
ah yes @martinjankowiak that is because the time zone of the virtual machine i used was ahead. perhaps the easiest thing to do would just be to update the readme? I think that's the only thing that is inconsistent

bkotzen commented:
this does make me think - the metadata is actually from 2024-01-01 (that was the most recent date available on GISAID). maybe it would be worth specifying the GISAID date in the metadata file moving forward?

martinjankowiak commented:
have you note been using the date that corresponds to the last day of data included? being the date you ran the inference is irrelevant

bkotzen commented:
right, agreed; I have been using the date of inference but I also have a record of the last day of data included so I can just change the names of the directories and push those changes in a separate PR

martinjankowiak commented:
ok updating the dates in a future PR would be good.

but what do you want to do this this PR? cuz as it stands the links the readme won't work

bkotzen committed:
inference results for 01.22.24

bkotzen committed:
Merge branch 'main' of github.com:bkotzen/bvas

bkotzen committed:
inference results for 01.22.24

bkotzen committed:
fix paper dates to align with metadata rather than run date

bkotzen committed:
update readme to align with corrected dates

bkotzen commented:
Hey @martinjankowiak a few updates here.
1. Added inference results for this week
2. Renamed the folders of previous results to align with the metadata date rather than the run date
3. Updated the README, in particular, I updated the links in the "more recent results" section

bkotzen commented:
And for the sake of record keeping, here are the changes:
(run date -> metadata date)
01-05 -> 01-01
05-18 -> 05-15
06-15 -> 06-13
06-30 -> 06-26
07-11 -> 07-10
08-10 -> 08-09
09-13 -> 09-04
09-20 -> 09-18
09-20-22 -> 09-19-22
11-03 -> 10-30
11-17 -> 11-13
11-29 -> 11-27
12-14 -> 12-11

martinjankowiak commented:
@bkotzen how sure about this are you? in particular the first run from august 2022, which i believe i did. i wrote the following in the readme:

> More recent inference results are to be found in directories named after the end date of the surveillance data used in the analysis, starting with August 10th: [08.10.22](https://github.com/bkotzen/bvas/blob/main/paper/08.10.22).

yet you seem to want to rename 08.10.22?

bkotzen commented:
Ok, I've combed through the commit history and found the commits which I have made and those which you have made. I can revert the dates back to how you had them for your commits!

bkotzen commented:
I believe this is the history and dates:

MARTIN 05-18
MARTIN 06-15
MARTIN 06-30
MARTIN 07-11
MARTIN 08-10
BEN 09-13 -> 09-04
BEN 09-20 -> 09-18
MARTIN 09-20-22
BEN 11-03 -> 10-28
BEN 11-17 -> 11-13
BEN 11-29 -> 11-27
BEN 12-14 -> 12-11

bkotzen committed:
inference results for 02.05.24

bkotzen committed:
inference results for 02.05.24

bkotzen commented:
Hey @martinjankowiak I've also added BVAS results for this week

bkotzen committed:
inference results for 02.23.2024

bkotzen committed:
inference results for 02.22.24

bkotzen commented:
here are more current results @martinjankowiak

bkotzen committed:
inference results for 03.04.24

bkotzen committed:
inference results for 03.04.24

bkotzen committed:
inference results for 03.21.24

bkotzen committed:
inference results for 03.22.24

bkotzen commented:
Typo in the commit name but folder name has the correct gisaid download date!

bkotzen committed:
inference results for 04.04.24

bkotzen committed:
inference results for 04.04.24

bkotzen committed:
inference results for 04.22.24

bkotzen committed:
inference results for 04.22.24

bkotzen committed:
inference results for 05.27.24

bkotzen committed:
inference results for 05.27.24

bkotzen committed:
05.27.24 run

bkotzen committed:
inference results for 06.24.24

bkotzen committed:
inference results for 06.24.24

bkotzen committed:
inference results for 07.22.24

bkotzen committed:
inference results for 07.22.24

bkotzen commented:
Hey @martinjankowiak just checking in - is this still a good place to be putting updated BVAS results?
